deployments/rialto/docker-compose.yml

```diff
@@ -145,3 +147,22 @@ services:
     ports:
       - "10133:9933"
       - "10144:9944"
+
+  prometheus-metrics:
+    image: prom/prometheus
+    volumes:
+      - ../../relays/ethereum/dashboard/prometheus.yml:/etc/prometheus/prometheus.yml
+    ports:
+      - "9090:9090"
+    depends_on: *all-nodes
+
+  grafana-dashboard:
+    image: grafana/grafana
+    volumes:
+      - ../../relays/ethereum/dashboard/grafana-datasource.yaml:/etc/grafana/provisioning/datasources/grafana-datasource.yaml
```

Comment:
I'd consider moving the entire `dashboard` directory somewhere closer to `deployments`. But I'm not super strong on this, cause having it nearby the `relay` code also is reasonable - so your call :)